eox_core/api/v1/views.py

```diff
@@ -54,6 +54,15 @@ def __init__(self, *args, **kwargs):
         self.query_params = None
         self.site = None
 
+    def initial(self, request, *args, **kwargs):
+        """
+        Loads the site into the object for every kind of request
+        """
+        super(UserQueryMixin, self).initial(request, *args, **kwargs)
+
+        if hasattr(request, 'site'):
+            self.site = request.site
```

Comment:
If request doesn't have site, what happens next? @felipemontoya

Reply:
If the request does not have a site, then None is the default value. It happens in many tests